doc/configuration.md

```diff
@@ -25,3 +25,4 @@ The following settings are provided:
   - `.` - Unsuccessful shrink
   - `!` - Successful shrink
 - `reproduce` - Opaque string that encodes the information necessary to reproduce minimal failures for properties. Since this string is opaque, it can only be obtained from a failed RapidCheck run. Refer to the [debugging documentation](debugging.md) for more information.
+- `shrink_tries` - Number of times each shrink is tested. Defaults to `1`.
```

Comment:
I've thinking a bit and I've realized that you could actually have the retrying built into the `runParallel` (or whatever it's called) instead. However, then you wouldn't be able to adjust it without rebuilding so maybe this is the best solution after all. What do you think?